proto/buf/alpha/registry/v1alpha1/plugin.proto

```diff
@@ -48,6 +48,8 @@ message PluginVersion {
   string plugin_name = 3;
   // The owner of the plugin to which this version relates.
   string plugin_owner = 4;
+  // The full docker image digest associated with this plugin version.
```

Comment:
Can you include a reference to the format of this string? It's probably somewhere in the Docker docs or OCI image spec.

Reply:
Added a link to https://github.com/opencontainers/image-spec/blob/main/descriptor.md#digests and clarified the comment.